code/pagetypes/Forum.php

```diff
@@ -107,16 +107,16 @@ function canModerate($member = null) {
 		if(!$member) $member = Member::currentUser();
 
 		if(!$member) return false;
-		
+
 		// Admins
-		if ($this->canEdit($member)) return true; 
+		if (Permission::checkMember($member, 'ADMIN')) return true;
```

Comment:
Why not have a `MODERATE_ANY_FORUM` permisson? What if you want to delegate forum moderators who don't have CMS permissions?